### PR TITLE
#109 add bike location

### LINKED ISSUE #109: Add location functionality in Add bike
fetch location from image and also have the user choose a location from a map if he or she wants to override

### PR DESCRIPTION
**Pull request by FredSoderberg**
#### What does this PR do?
Add location functionality to add bike. Both from image store, manual set and from image taken with camera
[#109](LINK_TO_ISSUE)

##### Why are we doing this? Any context or related work?
Location is necessary for our add bike functionality

#### Where should a reviewer start?
addbike.js
#### Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
closes #109

### CODE REVIEW

**Comment by FredSoderberg** [comment]
Backend has a reversegeo bug right now, frontend can solve this but they are working on it on their end :)

File: Bikeini/screens/AddBike.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 import { bindActionCreators } from 'redux';
 import PropTypes from 'prop-types';
 import RadioGroup from 'react-native-radio-buttons-group';
 import { Dropdown } from 'react-native-material-dropdown';
-import { ImagePicker, ImageManipulator } from 'expo';
+import { ImagePicker, ImageManipulator, Location } from 'expo';
 import permissions from '../utilities/permissions';
-import { headerStyle } from './header';
-
+import headerStyle from './header';
```
Comment (jakobemilandersson) [inline Bikeini/screens/AddBike.js]: Needs to be { headerStyle }

File: Bikeini/screens/Camera.js [hunk 2]
```diff
@@ -199,6 +200,7 @@ class CameraPage extends React.Component {
 
   setUriDetails = (photo) => {
     const { saveImageToState } = this.props;
+    console.log(photo);
```
Comment (jakobemilandersson) [inline Bikeini/screens/Camera.js]: Unnecessary console.log? 🐱

**Review by jakobemilandersson — APPROVED** [review]
Looks good! Great job! 😸

**Pull request merged**